INSERT INTO.sql

```diff
@@ -0,0 +1,52 @@
+INSERT INTO
+  sap_capire_bookshop_Books (
+    createdAt,
+    createdBy,
+    modifiedAt,
+    modifiedBy,
+    ID,
+    title,
+    descr,
+    author_ID,
+    genre_ID,
+    stock,
+    price,
+    currency_code,
+    image
+  )
+SELECT
+  coalesce(
+    ISO(value ->> '$."createdAt"'),
+    session_context('$now')
+  ),
+  coalesce(
+    value ->> '$."createdBy"',
+    session_context('$user.id')
+  ),
+  coalesce(
+    ISO(value ->> '$."modifiedAt"'),
```

Comment:
Having the converter around the `coalesce` simplifies the casting for typed databases. As the inpurt converter for Postgres is `cast` it will correct the `session_context` string type into the correct column type.